md/configure-client-of-bonita-bpm-engine.md

```diff
@@ -73,6 +75,7 @@ org.bonitasoft.engine.api-type = LOCAL
 #org.bonitasoft.engine.api-type = HTTP
 #server.url = http://localhost:8080
 #application.name = bonita
+#connections.max = 20
```

Comment:
:question: did you check it works with the Bonita Home client file configuration?